fenix/app/src/main/res/values/strings.xml

```diff
@@ -2058,6 +2058,156 @@
+    <!-- Description for grades D and F for review quality check adjusted grading. -->
+    <string name="review_quality_check_grade_d_f_description">Unreliable reviews</string>
+    <!-- Headline for review quality check review reliability card. -->
+    <string name="review_quality_check_grade_title">How reliable are these reviews?</string>
+    <!-- Headline for review quality check adjusted rating card. -->
+    <string name="review_quality_check_adjusted_rating_title">Adjusted rating</string>
+    <!-- Description for review quality check review reliability card. -->
+    <string name="review_quality_check_adjusted_rating_description">Unreliable reviews removed</string>
+    <!-- Headline for review quality check highlights card. -->
+    <string name="review_quality_check_highlights_title">Highlights from recent reviews</string>
+    <!-- Headline for review quality check explanation card. -->
+    <string name="review_quality_check_explanation_title">How we determine review quality</string>
+    <!-- Paragraph from review quality check explanation card explaining reliability. First parameter is the Fakespot product name. -->
+    <string name="review_quality_check_explanation_body_reliability">We use AI technology from %s by Mozilla to analyze the reliability of product reviews. This analysis will only help you assess review quality, not product quality.</string>
+    <!-- Paragraph from review quality check explanation card explaining review grading. First parameter is for highlighted string defined in review_quality_check_info_review_grade_header_highlighted. -->
+    <string name="review_quality_check_info_review_grade_header" tools:ignore="UnusedResources">We assign each product’s reviews a %s from A to F.</string>
+    <!-- Highlighted word from review quality check explanation card header. -->
+    <string name="review_quality_check_info_review_grade_header_highlighted" tools:ignore="UnusedResources">letter grade</string>
+    <!-- Description for review quality check grades A and B. -->
+    <string name="review_quality_check_info_grade_info_AB">Reliable reviews. We believe the reviews are likely from real customers who left honest, unbiased reviews.</string>
+    <!-- Description for review quality check grade C. -->
+    <string name="review_quality_check_info_grade_info_C">We believe there’s a mix of reliable and unreliable reviews.</string>
+    <!-- Description for review quality check grades D and F. -->
+    <string name="review_quality_check_info_grade_info_DF">Unreliable reviews. We believe the reviews are likely fake or from biased reviewers.</string>
+    <!-- Adjusted grading paragraph from review quality check explanation card. First parameter is for highlighted string defined in review_quality_check_explanation_body_adjusted_grading_highlighted. -->
+    <string name="review_quality_check_explanation_body_adjusted_grading">The %s is a based only on reviews we believe to be reliable.</string>
```

Comment:
Again, I'm not sure why we are using a parameter here instead of just one string?